### DISCUSSION TITLE
Issue when using rabbitMq and repeat_every from fast

### DISCUSSION THREAD

**Discussion by eveknow** [fastapi/fastapi#7356] [Questions]
Hi all.

I'm new with FAST API.

I try to implement example using FASTAPI:

1. Consumer to rabbitMQ
2. Run a schedule task.

My code below:

@app.on_event("startup")
def startup():
    # Rabbit Init
    # logging.info("start up")
    loop = asyncio.get_event_loop()
    asyncio.ensure_future(init_rabbit(loop))

@app.on_event("startup")
@repeat_every(seconds=1 * 60)
def remove_expired_tokens_task() -> None:
    print("Ab123")
    tracks = trackLongTimeRequestService.list()
    json = jsonable_encoder(tracks)
    print(json)

I got this errors:

INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
ERROR:    Task was destroyed but it is pending!
task: <Task pending name='Task-3' coro=<init_rabbit() running at /Users/thanhtk/Work/Others/github/python/fast-api/app/messages/rabbit.py:21> wait_for=<Task pending name='Task-5' coro=<Connection.connect() running at /Users/thanhtk/Work/Others/github/python/fast-api/venv/lib/python3.8/site-packages/aiormq/connection.py:237> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fdeee657250>()]> cb=[FutureStore.__on_task_done.<locals>.remover() at /Users/thanhtk/Work/Others/github/python/fast-api/venv/lib/python3.8/site-packages/aiormq/base.py:51, <TaskWakeupMethWrapper object at 0x7fdeee6225e0>()]>>
ERROR:    Task was destroyed but it is pending!
task: <Task pending name='Task-5' coro=<Connection.connect() running at /Users/thanhtk/Work/Others/github/python/fast-api/venv/lib/python3.8/site-packages/aiormq/connection.py:237> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fdeee657250>()]> cb=[FutureStore.__on_task_done.<locals>.remover() at /Users/thanhtk/Work/Others/github/python/fast-api/venv/lib/python3.8/site-packages/aiormq/base.py:51, <TaskWakeupMethWrapper object at 0x7fdeee6225e0>()]>

Anybody have the solution for this case.
Thanks all,

**Comment by tiangolo** [comment]
Please follow the new issue template. It's not possible to help you without it and without more info.

If you still have problems, please create a new issue following the template.
